crates/telio-model/src/features.rs

```diff
@@ -340,6 +340,9 @@ impl Default for FeatureValidateKeys {
 pub struct FeatureFirewall {
     /// Turns on connection resets upon VPN server change
     #[serde(default)]
+    pub neptun_reset_conns: bool,
+    /// Turns on connection resets upon VPN server change (Deprecated alias for neptun_reset_conns)
+    #[serde(default)]
     pub boringtun_reset_conns: bool,
```

Comment:
So normally app teams uses [FeatureConfigBuilder](https://github.com/NordSecurity/libtelio/blob/main/src/libtelio.udl#L459) instead of directly accessing members.

But because we kind of forgot to expose `boringtun_reset_conns` via that interface. Android had to use it via direct member access :/. So Yeah, good catch. 

In that case. I am resolving this conversation.